openjdk/ProblemList_openjdk8-openj9.txt

```diff
@@ -46,6 +46,7 @@ java/lang/invoke/LFCaching/LFMultiThreadCachingTest.java		https://github.com/ecl
 java/lang/invoke/LFCaching/LFSingleThreadCachingTest.java		https://github.com/eclipse/openj9/issues/1128	generic-all
 java/lang/invoke/LambdaFormTest.java		https://github.com/eclipse/openj9/issues/1128	generic-all
 java/lang/invoke/MethodHandles/TestCatchException.java		https://github.com/eclipse/openj9/issues/1128	generic-all
+java_lang_invoke_MethodHandlesProxiesTest   https://github.com/eclipse/openj9/issues/5950   generic-all
```

Comment:
Please update the format as java/lang/invoke/MethodHandlesProxiesTest.java

Reply:
ah, thank you for that catch!

Reply:
Apologies, please recheck